08-rails/how-to-use-postgres.md

```diff
@@ -0,0 +1,106 @@
+
+# Setting Up A Rails Project with Postgres
+
+Postgres (or PostgreSQL) is an open-source database which can provide scalability to your application.  By default Rails development sites use sqlite3 which can work fine, but it can be easier to use the same database application in development & production.  
+
+
+## Installing Postgres
+
+To install Postgres you can use the Brew package manager, then set brew to start postgres to launch every time the machine reboots and lastly install the pg gem to allow Ruby to talk to the database.  
+
+These steps you only need perform once.
+
+```bash
+$ brew install postgresql
+...
+$ brew services start postgresql
+...
+$ gem install pg
+...
+```
+
+## To setup a rails project with Postgres
+
+After Postgres is installed and configured you can create new rails apps with postgres this way.  `rails db:migrate` is an essential step because it sets up the required tables in the database for a successful start of the rails server.  
+
+```bash
+$ rails new my-rails-project -d postgresql
+...
+$ rails db:reset
+$ rails db:migrate
+...
+```
+
+
+## Converting Rails Apps Using Sqlite3
```

Comment:
Could be converting an old app is enough trouble / complication to not be worth covering.